package.json

```diff
@@ -62,6 +62,7 @@
     "coin:bitcoin": "pnpm --filter coin-bitcoin",
     "coin:evm": "pnpm --filter coin-evm",
     "coin:framework": "pnpm --filter coin-framework",
+    "coin:tester": "pnpm --filter coin-tester",
```

Comment:
Is it really necessary as it seems not to work without coin-module implement it?

Reply:
It's a helper for development so that we can run scripts inside the module like `pnpm coin:tester build` `pnpm coin:tester watch` directly from the root